modules/core/portal-target-platform-indexer-main/README.markdown

```diff
@@ -15,6 +15,7 @@ module_framework_base_dir=/home/rotty/AS/liferay-portal/osgi
 jars=(
 	${module_framework_base_dir}/core/com.liferay.portal.bootstrap.jar
 	${module_framework_base_dir}/static/com.liferay.portal.target.platform.indexer.jar
+	${module_framework_base_dir}/../tools/portal-target-platform-indexer-main/com.liferay.portal.target.platform.indexer.main.jar
```

Comment:
Hey @shuyangzhou,

Just wondering how will it work on ASes where the dxp_de.war will be deployed so any other than the Tomcat bundle. Will we distribute the tools in an archive or somewhere?